fraud/service.go

```diff
@@ -19,11 +18,15 @@ const (
 	fetchHeaderTimeout = time.Minute * 2
 )
 
+func init() {
+	Register(BadEncoding, UnmarshalBEFP)
+}
```

Comment:
Its fine to register it next to the service considering that BEFP is defined in the same pkg. However, current model allows anyone outside of the fraud pkg to register their own Fraud, so in that case the registering is better be close to the definition of the fraud type and maybe we should follow that approach.

Reply:
moved to Daser